env/env_posix.cc

```diff
@@ -778,6 +779,18 @@ class PosixEnv : public Env {
     return Status::OK();
   }
 
+  virtual Status CheckCompactionSize(const char *path_in_filesystem,
+      uint64_t size_added_by_compact) override {
+    // Figure out how much free space is in the filesystem
+    struct statvfs statbuf;
+    statvfs(path_in_filesystem, &statbuf);
+    uint64_t freespace = statbuf.f_bsize * statbuf.f_bfree;
+    if (freespace <= size_added_by_compact)
```

Comment:
we typically surround single-line conditional bodies with curly braces

Reply:
Gone now